serve.go

```diff
@@ -65,3 +73,14 @@ func serve(c Connector) error {
 		func() pconnector.DestinationPlugin { return NewDestinationPlugin(c.NewDestination()) },
 	)
 }
+
+// getGRPCTargetEnv checks the environmental variable provided by conduit-connector-protocol
+// and returns an error if it is not specified or empty.
+func getGRPCTargetEnv() (string, error) {
+	value := os.Getenv(pconduit.EnvConduitConnectorUtilitiesGRPCTarget)
+	if value == "" {
+		return "", fmt.Errorf("%q is not set. This indicates you are using an older version of Conduit."+
+			"Please, consider upgrading to at least version 0.11.0, available at https://github.com/ConduitIO/conduit/releases/tag/v0.11.0", pconduit.EnvConduitConnectorUtilitiesGRPCTarget)
```

Comment:
This message will get outdated by the time of the `v0.12.0` release. Can we rephrase somehow to indicate that the _minimum_ version is `v0.11.0`, but point them towards the latest release i.e. https://github.com/ConduitIO/conduit/releases/latest?

Reply:
@lovromazgon the "at least" should cover the minimum version, but pointing to latest is a good idea.